src/guess-game.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useState } from "react";
+
+export default function GuessGame(props) {
```

Comment:
You can do `export default function GuessGame({ cheatProbability }) {` to save you one line 😆
You will see this syntax a lot.

Reply:
Great Tip!